source/geometries/Next100Shielding.cc

```diff
@@ -36,46 +36,45 @@ namespace nexus {
     GeometryBase(),
 
     // Shielding internal dimensions
-    shield_x_ (155.  * cm),
-    shield_y_ (225.6 * cm),
-    shield_z_ (258.5 * cm), // 253.0 * cm before May 12, 2017
+    shield_x_ (158.  * cm),
```

Comment:
I thought we had decided to start using braces for initialisation: `shield_x_ {158. * cm}`